pkg/backup/restorer/restorer.go

```diff
@@ -160,3 +184,16 @@ func (br *BackupRestorer) cleanUp() error {
 
 	return nil
 }
+
+func fileExists(name string) bool {
+	_, err := os.Stat(name)
+	if err == nil {
+		return true
+	}
+	if errors.Is(err, os.ErrNotExist) {
+		return false
+	}
+
+	log.Warn().Err(err).Msg("file exists")
```

Comment:
Nit: maybe add `.Str("file_name", name)` to the chain to make this log line context more understandable.